pkg/kdump/kdump-view.jsx

```diff
@@ -263,12 +265,44 @@ export class KdumpPage extends React.Component {
     }
 
     handleTestSettingsClick() {
+        // if we have multiple targets defined, the config is invalid
+        const target = this.props.kdumpStatus.target;
+        let verifyMessage;
+        if (!target.multipleTargets) {
+            const path = target.path || "/var/crash";
+            if (target.type === "local") {
+                verifyMessage = fmt_to_fragments(
+                    ' ' + _("Results of the crash will be stored in $0 as $1, if kdump is properly configured."),
+                    <span className="ct-monospace">{path}</span>,
+                    <span className="ct-monospace">vmcore</span>);
+            } else if (target.type === "ssh") {
+                verifyMessage = fmt_to_fragments(
+                    ' ' + _("Results of the crash will be stored in $0 as $1, and also copied through SSH to $2, if kdump is properly configured."),
+                    <span className="ct-monospace">{path}</span>,
+                    <span className="ct-monospace">vmcore</span>,
+                    <span className="ct-monospace">{`${target.server}:${path}`}</span>);
+            } else if (target.type === "nfs") {
```

Comment:
These 7 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19107-20230815-214442-399a4d14-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)